src/managed/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs

```diff
@@ -68,6 +72,19 @@ private static string GetDarwinVersion()
             }
         }
 
+        private static string GetFreeBSDVersion()
+        {
+            String verson = RuntimeInformation.OSDescription;
+            try
+            {
+                return RuntimeInformation.OSDescription.Split()[1];
```

Comment:
I'm not familiar with the versioning story of FreeBSD, but is there any normalization that should happen here?

For example, with RedHat, RedHat guarantees that minor versions are backwards compatible.  This allows us to use 1 RID for RedHat 7.1, 7.2, 7.3, etc.

See https://github.com/dotnet/core-setup/blob/b5c917c0b24171ae27c7cfcfd2617b1088d41c11/src/managed/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs#L143-L150

If FreeBSD makes similar guarantees, we can simplify the RIDs used on that platform.  Ex. we could just have `FreeBSD.11` instead of both `FreeBSD.11.0` and `FreeBSD.11.1`.

Reply:
I don't know if it does, I can ask the FreeBSD community. I think as far as RID it should be be freebsd.11, The "display" Name should be FreeBSD. I would expect the OS Version presents true OS version.  
Would hat make sense?